<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createOrderButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Create order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateOrderButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Update order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listOrderButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>List orders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelOrderButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPrice">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="customerName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="userID">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="updateUserID">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="updateName">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="updateDate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="updatePrice">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Update User ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Price</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>25</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
